Front-End/index.html

```diff
@@ -2,18 +2,184 @@
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #00c774;">Active</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #00c774;">Active</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <!--/items-->
+                </tbody>
+
+            </table>
+        </div>
+    </div>
+    <div class="footer">
```

Comment:
The .footer and .copyright should be a under a footer tag. Read about semantic HTML.